engine/src/main/scala/org/scalarules/dsl/nl/ScalaRulesDslTrait.scala

```diff
@@ -1,16 +1,19 @@
 package org.scalarules.dsl.nl
 
 import org.scalarules.dsl.nl.datum.DatumImplicits
-import org.scalarules.dsl.nl.grammar.{DslCondition, DslConditionImplicits, DslEvaluationImplicits, GegevenWord}
+import org.scalarules.dsl.nl.grammar.{DslCondition, DslConditionImplicits, DslEvaluationImplicits, DslListFilterWord, DslLoopWordTrait, GegevenWord}
 import org.scalarules.engine._
 
 /**
   * Aggregates the keywords and implicit definitions of the Scala-Rules DSL. The implicits available in this
   * trait can be used by importing the `grammar` package object's members, or extending this trait.
   */
-trait ScalaRulesDsl extends DslConditionImplicits
+trait ScalaRulesDslTrait extends DslConditionImplicits
```

Comment:
Why did you add the term `Trait` to the name?

Compare these two usages: 

```scala
package object MyDsl extends ScalaRulesDsl

package object MyDsl extends ScalaRulesDslTrait
```

Long story short: I prefer the first one :)